File: FlightReportApp/src/app/Models/plane.ts
export class Plane {
    public id: number;
    public code: string;
    public color: string;
    public first_use_date: Date;

    constructor(id: number, code: string, color: string, first_use_date: Date) {
        this.id = id;
        this.code = code;
        this.color = color;
        this.first_use_date = first_use_date;
    }
}

File: FlightReportApp.API/Model/Plane.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace FlightReportApp.API.Model
{
	[Table("Plane")]
	public class Plane
	{
		public Plane()
		{
			Flights = new HashSet<Flight>();
		}

		public Plane(int id, string code, DateTime firstUseDate)
		{
			Id = id;
			Code = code;
			FirstUseDate = firstUseDate;
		}

		public int Id { get; set; }
		public string Code { get; set; }
		public DateTime? FirstUseDate { get; set; }

		[JsonIgnore]
		public virtual ICollection<Flight> Flights { get; set; }
	}
}
